rcl_yaml_param_parser/src/parse.c

```diff
@@ -151,7 +151,7 @@ void * get_value(
     style != YAML_DOUBLE_QUOTED_SCALAR_STYLE)
   {
     errno = 0;
-    ival = strtol(value, &endptr, 0);
+    ival = strtoll(value, &endptr, 0);
     if ((0 == errno) && (NULL != endptr)) {
       if ((NULL != endptr) && (endptr != value)) {
```

Comment:
I know this isn't directly related to what you are fixing here, but this `NULL != endptr` check is redundant; we can't get into this block if endptr == NULL.

```suggestion
      if (endptr != value) {
```

Reply:
@clalancette  Done.